test/XRP/fakes/fake-xrp-protobufs.ts

```diff
@@ -284,68 +309,118 @@ paymentAllFields.setDeliverMin(deliverMin)
 paymentAllFields.setSendMax(sendMax)
 paymentAllFields.setPathsList(pathList)
 
-// Transaction
+// TrustSets
+const trustSetMandatoryFields = new TrustSet()
+trustSetMandatoryFields.setLimitAmount(limitAmount)
+
+const trustSetAllFields = new TrustSet()
+trustSetAllFields.setLimitAmount(limitAmount)
+trustSetAllFields.setQualityIn(qualityIn)
+trustSetAllFields.setQualityOut(qualityOut)
+
+const trustSetSpecial = new TrustSet()
+trustSetSpecial.setLimitAmount(limitAmount)
+trustSetSpecial.setQualityIn(qualityInSpecial)
+trustSetSpecial.setQualityOut(qualityOutSpecial)
+
+// Transactions
 
 /**
  * Helper function to generate Transaction objects with the standard values from Payment objects.
  * There must be at most one of accountSet or payment.
  *
- * @param accountSet -AccountSet object to insert into the transaction.
- * @param payment -Payment object to insert into the transaction.
+ * @param transactionType - The type of transaction that is created.
+ * @param object - The object to be inserted into the transaction based on the transaction type.
  * @returns Payment Transaction with the included payment param.
+ * @throws Error if given bad data.
  */
 function buildStandardTestTransaction(
-  accountSet?: AccountSet,
-  payment?: Payment,
+  transactionType: Transaction.TransactionDataCase,
+  object: AccountSet | Payment | TrustSet,
```

Comment:
Nice, this'll scale. maybe `transactionData` or `additionalFields` would be a slightly more descriptive param